app/src/main/java/com/money/manager/ex/nestedcategory/NestedCategoryListFragment.java

```diff
@@ -228,7 +228,8 @@ public void onActivityCreated(Bundle savedInstanceState) {
         attachFloatingActionButtonToListView();
 
         // start loader
-        LoaderManager.getInstance(getActivity()).initLoader(ID_LOADER_NESTEDCATEGORY, null, this);
+//        LoaderManager.getInstance(getActivity()).initLoader(ID_LOADER_NESTEDCATEGORY, null, this);
```

Comment:
not understand why this statement does not work properly

Reply:
this is wired. what's the behavior here